### PR TITLE
ACMS-3361: Support SS v7.4: Enable the sitestudio_claro module by default for the customers.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes #ACMS-3361

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]

**Comment by rajeshreeputra** [comment]
Can we add below in root composer.json to make sure CI runs:
```
"extra":{
        "patches-ignore": {
            "drupal/acquia_cms_site_studio": {
                "acquia/cohesion": {
                    "3340269 - Supports Drush 12. If using Drush 10 or Drush 11 & this patch is failing, then ignore this patch": "https://www.drupal.org/files/issues/2023-06-13/3340269-support-drush-12-13.patch"
                }
            }
        }
}
```

**Comment by chandan-singh7929** [comment]
@deepakmishra2 - Can you please fix Static code analysis failure?
```
FILE: ...tarterkit/acquia-cms-starterkit/src/Helpers/Task/InstallTask.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 [247](https://github.com/acquia/acquia-cms-starterkit/actions/runs/7086361692/job/19284473312?pr=125#step:5:248) | ERROR | [x] Whitespace found at end of line
     |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by deepakmishra2** [comment]
> @deepakmishra2 - Can you please fix Static code analysis failure?
> 
> ```
> FILE: ...tarterkit/acquia-cms-starterkit/src/Helpers/Task/InstallTask.php
> ----------------------------------------------------------------------
> FOUND 1 ERROR AFFECTING 1 LINE
> ----------------------------------------------------------------------
>  [247](https://github.com/acquia/acquia-cms-starterkit/actions/runs/7086361692/job/19284473312?pr=125#step:5:248) | ERROR | [x] Whitespace found at end of line
>      |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
> ----------------------------------------------------------------------
> PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
> ----------------------------------------------------------------------
> ```

done

File: acms/acms.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ starter_kits:
     modules:
       require:
         - acquia_cms_site_studio
+        - acquia/cohesion:7.4.x-dev
```
Comment (chandan-singh7929) [inline acms/acms.yml:8]: @vishalkhode1 - Just a side note, we have to remove this once site studio 7.4 is released

File: acms/acms.yml [hunk 2]
```diff
@@ -17,13 +18,15 @@ starter_kits:
...
         - acquia_cms_site_studio
+        - sitestudio_claro
         - acquia_cms_page
         - acquia_cms_search
         - acquia_cms_tour
         - acquia_cms_toolbar
     themes:
       require:
         - acquia_claro
+        - acquia/cohesion-theme:7.4.x-dev
```
Comment (chandan-singh7929) [inline acms/acms.yml:29]: Same here

**Review by chandan-singh7929 — APPROVED** [review]

**Comment by vishalkhode1** [comment]
@deepakmishra2  Fix the failing PHPUnit tests:
FailingTest
Enable module headline is printing but we don't see any modules are getting enabled after that. Fix that as well.

**Pull request merged**